### PR TITLE
Add configuration related django-allauth

### PR DESCRIPTION
**Pull request by jbeezley**
Depends on https://github.com/girder/django-composed-configuration/pull/60.

I believe this works out of the box, but we could add more documentation regarding managing django-sites and social accounts or even just link out to allauth's documentation.

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This is the simplest configuration for developers to start with.
 1. Run `docker-compose run --rm django ./manage.py migrate`
 2. Run `docker-compose run --rm django ./manage.py createsuperuser`
    and follow the prompts to create your own user
+3. Run `docker-compose run --rm django ./manage.py collectstatic`
```
Comment (brianhelba) [inline {{ cookiecutter.project_slug }}/README.md:10]: Why is this necessary? The development server should be able to serve static files in-place?

Comment (jbeezley) [reply {{ cookiecutter.project_slug }}/README.md:10]: I thought so too, but it wasn't picking up the css file until after I ran that... Maybe something else was going on though.  I can remove it.

Comment (brianhelba) [reply {{ cookiecutter.project_slug }}/README.md:10]: If so, I'd consider that a bug. Let me know if it's still necessary, particularly after you delete the `staticfiles` directory entirely.
